From: Guido Guenther <agx@sigxcpu.org>
Date: Fri, 7 Mar 2008 09:08:26 +0100
Subject: Don't enable default network on boot

to not interfere with existing network configurations
---
 src/network/Makefile.inc.am | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/src/network/Makefile.inc.am b/src/network/Makefile.inc.am
index 23cf39b..ca516c3 100644
--- a/src/network/Makefile.inc.am
+++ b/src/network/Makefile.inc.am
@@ -87,8 +87,7 @@ install-data-network:
 	      $(DESTDIR)$(confdir)/qemu/networks/default.xml && \
 	    rm $(DESTDIR)$(confdir)/qemu/networks/default.xml.t; }
 	( cd $(DESTDIR)$(confdir)/qemu/networks/autostart && \
-	  rm -f default.xml && \
-	  $(LN_S) ../default.xml default.xml )
+	  rm -f default.xml )
 if WITH_FIREWALLD_ZONE
 	$(MKDIR_P) "$(DESTDIR)$(prefix)/lib/firewalld/zones"
 	$(INSTALL_DATA) $(srcdir)/network/libvirt.zone \
